cardano_node_tests/tests/tests_conway/test_conway_extras.py

```diff
@@ -0,0 +1,925 @@
+        )
+        reqc.cli090.start(url=helpers.get_vcs_link())
+        # Obtain drep registration certificate
+        reg_cert = cluster.g_conway_governance.drep.gen_registration_cert(
+            cert_name=temp_template,
+            deposit_amt=deposit_amt,
+            drep_vkey_file=drep_keys.vkey_file,
+            drep_metadata_url=drep_metadata_url,
+            drep_metadata_hash=drep_metadata_hash,
+            destination_dir=".",
+        )
+        tx_files_reg = clusterlib.TxFiles(
+            certificate_files=[reg_cert],
+            signing_key_files=[payment_addr.skey_file, drep_keys.skey_file],
+        )
+
+        # Submit drep registration certificate
+        clusterlib_utils.build_and_submit_tx(
+            cluster_obj=cluster,
+            name_template=f"{temp_template}_reg",
+            src_address=payment_addr.address,
+            submit_method=submit_method,
+            use_build_cmd=use_build_cmd,
+            tx_files=tx_files_reg,
+            deposit=deposit_amt,
+        )
+
+        # Wait for some blocks and again submit drep registration certificate
+        cluster.wait_for_new_block(new_blocks=2)
+
+        # For handling case where str method of submit api error returns extra \\
+        error_string_content = ""
+
+        # Expecting error for both cases as drep is already registered
+        if submit_method == "api":
+            with pytest.raises(submit_api.SubmitApiError) as excinfo_submit:
+                clusterlib_utils.build_and_submit_tx(
+                    cluster_obj=cluster,
+                    name_template=f"{temp_template}_ret",
+                    src_address=payment_addr.address,
+                    submit_method=submit_method,
+                    use_build_cmd=use_build_cmd,
+                    tx_files=tx_files_reg,
+                    deposit=deposit_amt,
+                )
+            error_string_content = "\\"
+        else:
+            with pytest.raises(clusterlib.CLIError) as excinfo_cli:
+                clusterlib_utils.build_and_submit_tx(
+                    cluster_obj=cluster,
+                    name_template=f"{temp_template}_ret",
+                    src_address=payment_addr.address,
+                    submit_method=submit_method,
+                    use_build_cmd=use_build_cmd,
+                    tx_files=tx_files_reg,
+                    deposit=-deposit_amt,
+                )
+
+        drep_id = cluster.g_conway_governance.drep.get_id(
+            drep_vkey_file=drep_keys.vkey_file,
+            out_format="hex",
+        )
+        err_msg = str(excinfo_submit.value if excinfo_submit else excinfo_cli.value)
+        assert (
+            f'ConwayDRepAlreadyRegistered (KeyHashObj (KeyHash {error_string_content}"{drep_id}'
+            in err_msg
+        ), err_msg
+        reqc.cli090.success()
+
+    @allure.link(helpers.get_vcs_link())
+    @pytest.mark.testnets
+    @pytest.mark.smoke
+    def test_deposit_return_committee(
```

Comment:
we can use a pre existent test to check this - like one of `test_commitee.py` file

Reply:
Thank you for the feedback. The deposit return test after enactment has been updated to `test_add_rm_committee_members` function.